pkg/payload/outgoing/payload.go

```diff
@@ -0,0 +1,16 @@
+package outgoing
+
+type (
+	Payload struct {
```

Comment:
Just repeating this for the sake of not missing it :)

I would prefer if we keep this pkg/payload/... removed - it's an obsolete pattern that was part of messaging components and and old mindset. If you really really need it, move it under websocket package. But really.. let's discuss this on a call and find a better way to solve this.